src/batou_ext/versions.py

```diff
@@ -27,108 +27,103 @@ def get_words():
     return ThreadedCompleter(FuzzyWordCompleter(get_words))
 
 
-class Updater:
-    """Version update, interactive or automatic."""
-
-    environment_name: str = None
-    _environment = None
-    _target_versions = None
-
-    def __init__(self, basedir):
-        self.basedir = basedir
-
-    @property
-    def environment(self):
-        if self._environment is None:
-            assert self.environment_name
-            self._environment = batou.environment.Environment(
-                self.environment_name
-            )
-            self._environment.load()
-        return self._environment
-
-    def update_from_branch(self, branch: str):
-        envs = sorted(os.listdir(os.path.join(self.basedir, "environments")))
-        for env_name in envs:
-            env = batou.environment.Environment(env_name)
-            env.load()
-            if env.branch == branch:
-                print(f"Updating environment: {env_name}")
-                self.environment_name = env_name
-                self.set_versions()
-
-        if not self.environment_name:
-            raise ValueError(
```

Comment:
This check to make sure anythign got updated on $branch is missing in the new code AFAIS.
Is that intended?

Reply:
might be useful to add it, sure